docs/send-data/hosted-collectors/microsoft-source/azure-metrics-source.md

```diff
@@ -0,0 +1,99 @@
+---
+id: azure-metrics-source
+title: Azure Metrics Source
+sidebar_label: Azure Metrics Source
+description: Learn how to collect Azure metrics.
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/azure-metrics/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/azure-metrics/example.json';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/send-data/azure-event-hub.svg')} alt="icon" width="40"/>
+
+Azure Metrics Source is used to collect all available metrics from Azure Monitor into Sumo Logic. To obtain a complete list of metrics that are collected using this source, refer to the [Azure Documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/metrics-index#supported-metrics-and-log-categories-by-resource-type).
+
+## Setup
+
+### Vendor configuration
+
+The Azure Metrics Source requires you to provide **Tenant Id**, **Client Id**, and **Client Secret** while configuring. To obtain these values, follow the below steps:
+
+1. Sign in to the [Azure portal](https://portal.azure.com/#create/Microsoft.Template/uri/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FRoleAssignmentSubscriptions.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FCreateUIDefSubscriptions.json).
```

Comment:
Maisie just gave feedback that we can probably separate this into two steps:
```
1. Sign in to the [Azure portal](https://portal.azure./com)
2. Navigate to Sumo Logic's [Custom Deployment wizard](https://portal.azure.com/#create/Microsoft.Template/uri/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FRoleAssignmentSubscriptions.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FSumoLogic%2Fsumologic-azure-serverless%2Fmain%2FAzureMetrics%2FCreateUIDefSubscriptions.json).
3. Under the **Basics** tab, ...
```

Reply:
Typo: `https://portal.azure.com`

Reply:
updated!